todo_app/app.py

```diff
@@ -48,17 +60,20 @@ def add_item():
         todo_item_status = request.form.get('status')
         # mongo.addItemstoMongo({'cardid':todo_item_id})
         mongo.addItemstoMongo(todo_item_id, todo_item_title, todo_item_status)
+        app.logger.info("added items to Mongo")
         return redirect(url_for("index"))
 
     
     @app.route('/deletecard', methods=["POST"])
     def deleteonecard():
         mongo.deleteOneCard(request.form.get('id'))
+        app.logger.info("deleted one item from Mongo")
         return redirect(url_for("index"))
         
 
     @login_manager.user_loader
     def load_user(user_id):
+        app.logger.info("user loaded")
```

Comment:
This is not a very useful message. It will appear in your logs every time a logged in user makes any request at all.

And you're not even logging their ID. So this will just clutter your logs with a lot of messages equivalent to "someone did something".